composer.json

```diff
@@ -43,18 +41,22 @@
         "symfony/validator": "^4.4 || ^5.0",
         "symfony/yaml": "^4.4 || ^5.0",
         "webmozart/assert": "^1.8",
-        "willdurand/hateoas-bundle": "^2.0",
         "winzou/state-machine-bundle": "^0.3.2 || ^0.4.3 || ^0.5"
     },
     "replace": {
         "sylius/resource": "self.version"
     },
     "conflict": {
-        "amphp/amp": "^2.4.3"
+        "amphp/amp": "^2.4.3",
+        "friendsofsymfony/rest-bundle": "<3.0",
```

Comment:
I wonder whether we shouldn't also add conflict for the next major version (to be removed when we decide to support it). Then it would be `<3.0 >=4.0` (and similar for other requirements here). What do you think?